src/SUMMARY.md

```diff
@@ -207,6 +207,9 @@
   - [Mutable Static Variables](unsafe-rust/mutable-static.md)
   - [Unions](unsafe-rust/unions.md)
   - [Unsafe Functions](unsafe-rust/unsafe-functions.md)
+    - [Unsafe External Functions](unsafe-rust/unsafe-extern-c-functions.md)
+    - [Unsafe Rust Functions](unsafe-rust/unsafe-rust-functions.md)
+    - [Calling Unsafe Functions](unsafe-rust/calling-unsafe-functions.md)
```

Comment:
For subslides, we've used a subdirectory to contain the subslides. Also, only the top-level slide has the timing information. See src/methods-and-traits/traits for an example.